tools/common.c

```diff
@@ -158,6 +166,51 @@ struct poptOption connect_options[] = {
 #endif /* WITH_SSL */
     {NULL, '\0', 0, NULL, 0, NULL, NULL}};
 
+void read_authfile(const char *path)
+{
+  size_t n;
+  FILE *fp;
```

Comment:
Initialize this to NULL (this should silence warnings about potential use of uninitialized fp (see scan-build check),

Reply:
Done.